### PR TITLE
Update Build dApps section

### PR DESCRIPTION
**Pull request by shrinathprabhu**
Resolves [AR-2630](https://team-1624093970686.atlassian.net/browse/AR-2630).

## Changes
- Update tabs in build dapps section
- Update styling of build-usecases component
- Change devrel@arcana.network to support@arcana.network
- Remove BuildOverview component

## Screenshots
**Before**
Screenshot from 2022-05-25 12-53-42
Screenshot from 2022-05-25 12-54-28

**After**
Screenshot from 2022-05-25 12-48-58
Screenshot from 2022-05-25 12-51-32

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

**Review by karthik-durai — APPROVED** [review]

File: components/BuildSection.vue [hunk 1]
```diff
@@ -66,31 +63,4 @@ section {
...
-    margin-left: -5%;
-  }
-}
-
-.build-illustration {
-  margin-top: -10rem;
-  z-index: -1;
-  background-image: url('~assets/images/testnet-build-top-gradient.png'),
-    url('~assets/images/testnet-build-bottom-gradient.png'),
-    url('~assets/images/testnet-build.png');
```
Comment (ajithranka) [inline components/BuildSection.vue:85]: Can we remove these images too?

File: components/BuildUseCases.vue [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
     <v-stack
       v-for="useCase in useCasesList"
       :key="useCase.key"
       direction="row"
       sm-direction="column"
-      gap="1.875rem"
-      sm-gap="1.5rem"
-      justify="space-between"
+      gap="10rem"
+      sm-gap="1rem"
```
Comment (ajithranka) [inline components/BuildUseCases.vue:9]: Why did we increase the gap here and remove `justify="space-between"`?

Comment (shrinathprabhu) [reply components/BuildUseCases.vue:9]: It was adding too much space in larger screens, ref: Screenshot from 2022-05-25 12-53-42

Comment (shrinathprabhu) [reply components/BuildUseCases.vue:9]: By adding gap: 10rem the spacing becomes less
Screenshot from 2022-05-25 12-51-32

**Review by ajithranka — APPROVED** [review]

**Pull request merged**